### PR TITLE
[KlaudSol-CMS-#175] Should display the user-friendly content type name, not the slug

### PR DESCRIPTION
**Pull request by Norlant1**
friendly type content

### CODE REVIEW

File: components/Util.js [hunk 1]
```diff
@@ -71,9 +71,13 @@ export const sortByOrderAsc = (first, second) => first[1].order - second[1].orde
 export const isNumber = (str) => {
   return !isNaN(str);
 }
-  
-
 
+export const findContentTypeName = (arr,slugName) => (
+  arr.find((obj)=> (
```
Comment (angeli-mercedes) [inline components/Util.js:76]: If there is only one parameter inside the `find` function, you don't have to put parenthesis.
```suggestion
  arr.find( obj => (
```

File: pages/admin/content-manager/[entity_type_slug].js [hunk 2]
```diff
@@ -110,6 +100,7 @@ export default function ContentManager({ cache }) {
     dispatch({type: PAGE_SETS_RENDERER,payload: defaultPageRender});
   }, [entity_type_slug]);
 
+  console.log()
```
Comment (angeli-mercedes) [inline pages/admin/content-manager/[entity_type_slug].js:103]: Kindly remove this `console.log()` line. Thanks!

**Review by angeli-mercedes — CHANGES_REQUESTED** [review]
Kindly see comments. Thanks @Norlant1 ! :)

**Review by angeli-mercedes — APPROVED** [review]

File: components/elements/inner/ContentManagerSubMenu.js [hunk 3]
```diff
@@ -59,11 +59,10 @@ const ContentManagerSubMenu = ({title, currentTypeSlug}) => {
     useEffect(() => { 
       (async () => {
 
-        await loadEntityTypes({rootState, rootDispatch, onStartLoad, onEndLoad});
+        await loadEntityTypes({rootState, rootDispatch, onStartLoad, onEndLoad, currentTypeSlug});
```
Comment (ardeearam) [inline components/elements/inner/ContentManagerSubMenu.js:62]: `loadEntityTypes`, as the name suggests, is intended to load all of the entity types from the database. Adding an extra parameter makes this function deviate from its original intention. I think the handling of the current slug can be handled differently (see other code comments).

**Pull request merged**